accumulator/starter/main.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
```

Comment:
Usually stdlib imports are grouped together and modern autoformat fixes this usually